clang/lib/Sema/SemaTemplateVariadic.cpp

```diff
@@ -38,7 +38,7 @@ class CollectUnexpandedParameterPacksVisitor
   unsigned DepthLimit = (unsigned)-1;
 
 #ifndef NDEBUG
-    bool ContainsIntermediatePacks = false;
+  bool ContainsIntermediatePacks = false;
```

Comment:
We gain nothing by hiding that behind NDEBUG; we can afford a boolean on the stack